Source/WebCore/Modules/mediastream/RTCEncodedFrame.h

```diff
@@ -40,6 +40,8 @@ namespace WebCore {
 
 class RTCEncodedFrame : public RefCounted<RTCEncodedFrame> {
 public:
+    virtual ~RTCEncodedFrame() { }
```

Comment:
I don't think this is necessary. The constructor is marked as protected so no-one can destroy a RTCEncodedFrame directly. Having subclasses is fine as long as their destructor gets called first, which it would since the base class constructor is not public.

Reply:
Or maybe it is necessary because of the way RefCounted calls `delete`? I have to look more into it.

Reply:
You can't construct the base class, but you can 
* construct the subclass
* cast the subclass pointer to the base class pointer
* delete the base class pointer

In this case, I think that can mean that you fail to destroy RTCEncodedVideoFrame::m_metadata::dependencies (which is a Vector).